ui/plugins/controller.js

```diff
@@ -339,7 +339,20 @@ treeherder.controller('PluginCtrl', [
                    ($scope.job.state === "pending" || $scope.job.state === "running");
         };
 
-        $scope.retriggerJob = function (jobs) {
+        $scope.spinButton = function (selector, spinclass) {
+            $( selector ).removeClass( spinclass );
```

Comment:
Stole the spacing from https://github.com/mozilla/treeherder/blob/e53a3393a2783ec878d3f9c2e55997d42a98458d/ui/plugins/pinboard.js#L48

Happy to remove the spaces.